assets/textures/menu_backgrounds/SphereSkyMaterial_Base_color.png.import

```diff
@@ -2,21 +2,20 @@
 
 importer="texture"
 type="CompressedTexture2D"
-uid="uid://blntn0acli076"
-path.s3tc="res://.godot/imported/SphereSkyMaterial_Base_color.png-5187083fa936831c9a77057c5ae3c37b.s3tc.ctex"
+uid="uid://cxfwmrejyt75q"
+path="res://.godot/imported/SphereSkyMaterial_Base_color.png-5187083fa936831c9a77057c5ae3c37b.ctex"
 metadata={
-"imported_formats": ["s3tc_bptc"],
-"vram_texture": true
+"vram_texture": false
```

Comment:
Why is this texture no longer compressed? This will take a ton of more disk space and memory when loaded now.
If for visual quality it is absolutely necessary that's fine, but I'd first try tweaking the compression options to see if some of them would be good enough without using as much memory as an uncompressed texture.

Reply:
I think that change was made randomly by Godot. I don't recall doing anything that should have affected that file, but it started being flagged as changed, and I didn't realize that was why. I'll change it back.

Reply:
Godot keeps reverting this whenever I change it. Even without it loaded in a scene.

Reply:
What do you mean by reverting it? You need to use the Godot import options panel to tweak the options back to what works for one used in a 3D scene. If you are just trying to revert this single line then it is likely incorrect as it depends on other settings that also must be changed at the same time.

Reply:
I've re-imported it to revert line 9, but not 8. I can import it as s3tc_bptc with the path endings changed from .ctex to .s3tc.ctex. I'm not sure which is better.

Reply:
That path ending if I remember correctly is just what Godot 4 now does, so the import paths changing to whatever godot wants is fine as long as the compression / other options are correct. The import path is just some godot internal stuff we don't need to worry about at all.